recipe/meta.yaml

```diff
@@ -22,11 +22,12 @@ requirements:
   host:
     - pip
     - python
-    # bootstrapping
-    - packaging >=21.3
-    - pathspec >=0.10.1
-    - pluggy >=1.0.0
-    - tomli >=2.0
+    - wheel
```

Comment:
`wheel` shouldn't be required. `hatchling` uses its own code to package itself, see https://github.com/pypa/hatch/blob/master/backend/pyproject.toml#L3-L4.